.github/dependabot.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: /
+    schedule:
+      interval: daily
+    allow:
+      # Internal gems
```

Comment:
Should we add slimmer here?

Reply:
Seems reasonable 👍 under `# Framework gems` you mean?

Reply:
I think either should be fine? It's an internal framework gem I guess

Reply:
If we add it here we should add it to all the other repos though

Reply:
Oh, [you're right](https://github.com/alphagov/slimmer) - I was confusing it with [SlimerJS](https://slimerjs.org/)!

Reply:
I've now updated the open PRs where this is relevant, and opened some patch PRs where this was originally missed:

https://github.com/alphagov/email-alert-frontend/pull/842
https://github.com/alphagov/info-frontend/pull/721
https://github.com/alphagov/feedback/pull/1051
https://github.com/alphagov/collections/pull/1833
https://github.com/alphagov/travel-advice-publisher/pull/925
https://github.com/alphagov/smart-answers/pull/4685
https://github.com/alphagov/finder-frontend/pull/2152

Thanks for spotting!

Reply:
(And now this repo!)